components/discord_bot/sources/common.mjs

```diff
@@ -0,0 +1,16 @@
+import constants from "../constants.mjs";
+import common from "../common.mjs";
+
+export default {
+  props: {
+    ...common.props,
+  },
+  methods: {
+    _getLastMessageIDs() {
```

Comment:
Include all of `common` here, and you can leave out the `props` section. Also, don't forget to include common.methods here.
```
export default {
  methods: {
    ...common.methods,
    _getLastMessageIDs() {
```